app/api/login.js

```diff
@@ -0,0 +1,8 @@
+import client from "./client";
+
+const login = (username, password) =>
+  client.post("/login", { username, password });
+
```

Comment:
#### Non-blocking comment
It is okay for now but we should also add validation logic before service call. 
For eg: validate username and password are not null, regex (if any) etc...

-> This will help to prevent service call for which backend will throw error.

Reply:
Yes, will create a separate dedicated PR for validation.